config/locales/ptbr.yml

```diff
@@ -59,6 +59,7 @@ ptbr:
      location: "Localização"
      description: 'Descrição'
      error_explanation: 'Please fill out the marked fields!'
+     press_enter: 'Pressione ENTER para adicionar'
    time_ago: "%{date} atrás"
    write_to_us: "join.ifme@gmail.com"
```

Comment:
Ahh good question and catch, the reason why we have this is that the English sentence did not match structurally with the Spanish and Portuguese translations. Hmm maybe write_to_us can be renamed to something else then.